addons/mail/static/src/views/web/activity/activity_controller.scss

```diff
@@ -63,42 +63,46 @@
     }
 
     // it contains a kanban card representing the record
-    .o_activity_record > div {
-        display: flex;
-        flex: 1 1 auto;
-        align-items: center;
+    .o_activity_record {
+        > div {
+            display: flex;
+            flex: 1 1 auto;
+            align-items: center;
 
-        .o_m2o_avatar > img, > img {
-            object-fit: cover;
-            width: 32px;
-            height: 32px;
-            max-height: 32px;
-        }
-
-        .o_m2m_avatar, .o_m2m_avatar_empty {
-            width: 32px !important;
-            height: 32px !important;
-        }
-        .o_m2o_avatar {
-            margin-right: 16px;
-        }
-        .o_m2o_avatar .o_delete {
-            font-size: initial !important;
-        }
-        .o_m2m_avatar_empty {
-            padding-top: 2px;
-            font-size: large;
-        }
+            .o_m2o_avatar > img, > img {
+                object-fit: cover;
+                width: 32px;
+                height: 32px;
+                max-height: 32px;
+            }
+            .o_m2m_avatar, .o_m2m_avatar_empty {
+                width: 32px !important;
+                height: 32px !important;
+            }
+            .o_m2o_avatar {
+                margin-right: 16px;
+            }
+            .o_m2o_avatar .o_delete {
+                font-size: initial !important;
+            }
+            .o_m2m_avatar_empty {
+                padding-top: 2px;
+                font-size: large;
+            }
 
-        .o_text_bold {
-            font-weight: $font-weight-bold;
+            .o_text_bold {
+                font-weight: $font-weight-bold;
+            }
+            .o_text_block {
+                @include o-text-overflow($max-width: 15rem);
+                display: block;
+            }
         }
-
-        .o_text_block {
-            @include o-text-overflow($max-width: 15rem);
-            display: block;
+        &:hover {
+            background-color: $component-active-bg;
         }
     }
+    
```

Comment:
Thank you for the PR!

All the above looks like 90% whitespace diff though, which makes it more difficult to 1) see what actually changed here without tweaking settings, and 2) follow git blame as there'll be an additional commit to browse. 

Do we achieve the same by only add these 3 spot on lines of yours instead?

```scss
    .o_activity_record:hover {
        background-color: $component-active-bg;
    }
```

Reply:
Understandable! @flch-odoo (Apologies for the mess : /)
Updated the PR :+1: